scrapysplashwrapper/crawl.lua

```diff
@@ -27,6 +27,15 @@ function main(splash, args)
     -- Set a reasonable viewport
     splash:set_viewport_size(1280, 800)
 
+    splash:on_request(function(request)
+        request:set_proxy{
+            host = args.myproxy['host'],
+            port = args.myproxy['port'],
+            username = '',
+            password = '',
```

Comment:
As we have the option, it would be nice to allow a user to also pass a username/password. Some proxies will require it.

Reply:
True, completely ommited this one